scripts/manage_org.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/bash
+
+#==============================================================================
+#title           :mamage_org.sh
+#description     :This script is a compilation of all the commonly used CCI commands
+#usage           :Under Help
+#=======================================
+
+function help {
+    echo "**This is script written to help QE manage scratch org either for testing or regression purpose **"
+    echo "Below Options can be provided:"
+    echo ' \t '"
+           -o        org to base the new scratch org from
+           -n        new orgname to be created
+           -f        flowname to be applied [ regression/dev/qa]
+           -d        active days
+           -a        action to perform [create/delete/list/info ]
+           -p        additional actions to create an additional usre or to enable rd2 within the org [createuser/rd2] "
+
+    echo "Create a new org or rebuild an org :     ./manage_scratchorg.sh [-o org] [-n orgname] [-f feature] [-d days] [-a create]"
```

Comment:
can we have a uniform file name across the script? I see manage_scratchorg.sh in some and manage_org.sh in some